Training25/Program.cs

```diff
@@ -4,10 +4,31 @@
 // ------------------------------------------------------------------
 // Program.cs
 // Program on main branch.
+// T4: Program to print chess board.
 // ------------------------------------------------------------------------------------------------
+using System.Text;
+using static System.Console;
 namespace Training25;
+
 internal class Program {
-   static void Main (string[] args) {
-      Console.WriteLine ("Hello, World!");
+   static void Main () {
+      OutputEncoding = new UnicodeEncoding ();
+      WriteLine ("┏━━━┳━━━┳━━━┳━━━┳━━━┳━━━┳━━━┳━━━┓");
+      PrintRow ("♜♞♝♛♚♝♞♜");
+      char[] pawns = { '♟', '♙' };
+      for (int i = 0; i < 13; i++)
+         // In even iterations print line separator.
+         if ((i & 1) == 0) WriteLine ("\n┣━━━╋━━━╋━━━╋━━━╋━━━╋━━━╋━━━╋━━━┫");
+         else {
+            char rChar = ' ';
+            // Print black or white pawns for i = 1 or i = 11 respectively.
+            if (i % 10 == 1) rChar = pawns[0 + (i / 10)];

```

Comment:
```suggestion
            if (i % 10 == 1) rChar = pawns[0 + (i / 10)];
```
Why is 0 being added to i/10? There will be no change in the value anyways.

Use ternary operator instead of `if`
```suggestion
            char rChar = i % 10 == 1 ? pawns[i / 10] : ' ';
```